IMAGES.rst

```diff
@@ -246,19 +246,21 @@ Images with a commit SHA (built for pull requests and pushes)
 
 .. code-block:: bash
 
-  ghcr.io/apache/airflow-<BRANCH>-pythonX.Y-ci-v2:<COMMIT_SHA>    - for CI images
-  ghcr.io/apache/airflow-<BRANCH>-pythonX.Y-v2:<COMMIT_SHA>       - for production images
-  ghcr.io/apache/airflow-<BRANCH>-pythonX.Y-build-v2:<COMMIT_SHA> - for production build stage
-  ghcr.io/apache/airflow-python-v2:X.Y-slim-buster-<COMMIT_SHA>   - for base Python images
+  ghcr.io/apache/airflow/<BRANCH>/ci/python<X.Y>:<COMMIT_SHA>         - for CI images
+  ghcr.io/apache/airflow/<BRANCH>/prod/python<X.Y>:<COMMIT_SHA>       - for production images
```

Comment:
I'm glad they enabled this, it's so much nicer

Reply:
indeed. Even if it's "sort of done" - only the first "/" is real separator - the others are just part of the name and translated to %2F in URL :D. The URLs how to get the packages look weird:

https://github.com/apache/airflow/pkgs/container/airflow%2Fv2-1-test%2Fprod-build%2Fpython3.8

Reply:
The funny thing is that in Github UI tje first %2F in the name is treated as separator, even if it is part of the name :


![Screenshot from 2021-08-05 18-29-26](https://user-images.githubusercontent.com/595491/128387679-db18388f-b454-43ea-b149-5cdc851f1ec8.png)